src/frontend/packages/lib_video/src/components/common/VideoWidgetProvider/widgets/SharedLiveMedia/SharedLiveMediaItem/ToggleDownload/index.tsx

```diff
@@ -19,26 +18,26 @@ const messages = defineMessages({
     description: 'Message displayed when shared media update has failed.',
     id: 'component.AllowedDownloadButton.updateSharedLiveMediaFail',
   },
-  buttonLabel: {
-    defaultMessage:
-      'Click on this button to stop allowing students to download this media.',
+  labelToggleAllowDownloadSupport: {
+    defaultMessage: 'Allow download',
     description:
-      'The label of the button for stopping to allow students to download the media.',
-    id: 'component.AllowedDownloadButton.buttonLabel',
+      'Label of the toggle allowing to download the support share by the instructor',
+    id: 'component.StartSharingButton.labelToggleAllowDownloadSupport',
```

Comment:
as you rename this component, you should change all the id for all messages. And I think it should not be `StartSharingButton` but `ToggleDownload`.